setup.py

```diff
@@ -2,6 +2,23 @@
 
 import sys
 from setuptools import find_packages, setup
+import git
+from git import Repo
```

Comment:
I think you can strike the `import git` line, since `from git import Repo` should cover it.